src/components/SearchBar/SearchBar.js

```diff
@@ -89,6 +89,16 @@ export default class SearchBar extends React.Component {
      * with "selected" items moved to the start of the scope items array.
      */
     sortItems: PropTypes.func, // eslint-disable-line react/require-default-props
+
+    /**
+     * Provide accessible label text for the scopes MultiSelect.
+     */
+    titleText: PropTypes.string,
+
+    /**
+     * Whether or not the scopes MultiSelecto label is visually hidden.
+     */
+    hideScopesLabel: PropTypes.bool,
```

Comment:
Should these be knobs in the storybook previews?

Reply:
Thanks! Just added it 👍